packages/block-library/src/tag-cloud/style.scss

```diff
@@ -18,4 +18,18 @@
 		margin-left: 5px;
 		text-decoration: none;
 	}
+
+	&.is-style-pill a {
+		border: 1px solid var(--wp--preset--color--foreground);
+		border-radius: 5px;
+		font-size: var(--wp--preset--font-size--normal);
+		margin-bottom: 10px;
+		padding: 5px 10px;
+
+		&:active,
+		&:hover {
+			background-color: var(--wp--preset--color--foreground);
+			color: var(--wp--preset--color--background);
```

Comment:
These variables don't exist by default, unfortunately. 

I'm not sure that we can specify a nice reversed-out hover state without tackling https://github.com/WordPress/twentytwentytwo/issues/99 first.

Reply:
I think we can avoid hover effects here